api/hypershift/v1beta1/hostedcluster_types.go

```diff
@@ -1911,6 +1911,44 @@ type OpenStackPlatformSpec struct {
 	// +listType=set
 	// +optional
 	Tags []string `json:"tags,omitempty"`
+
+	// IngressFloatingIP is an available floating IP in your OpenStack cluster that will
+	// be associated with the OpenShift ingress port.
+	//
+	// +optional
+	IngressFloatingIP string `json:"ingressFloatingIP,omitempty"`
+
+	// IngressProvider specifies the ingress provider to use for the cluster.
```

Comment:
I guess I could have add more docs on this one.
I'll do it in a next PR if people don't mind, unless I'm asked to change something else, then I'll do it in batch.
Thanks